python/lsst/skymap/patchInfo.py

```diff
@@ -50,7 +50,8 @@ class PatchInfo:
         Patch sequential index
     tractWcs : `lsst.afw.geom.SkyWcs`
         Tract WCS object.
-    cellInnerDimensions : `Iterable` [`int`, `int`] or `lsst.geom.Extent2I`, optional
+    cellInnerDimensions : `Iterable` [`int`, `int`] or `lsst.geom.Extent2I`, \
```

Comment:
```suggestion
    cellInnerDimensions : `~collections.abc.Iterable` [`int`] or `lsst.geom.Extent2I`, \
```

Reply:
Thanks. Not sure how I missed that one, and again I think they might be trying to tell us that it's an iterable of list. `Tuple[int, int]` might be clearer.